Jenkinsfile

```diff
@@ -85,23 +85,24 @@ pipeline {
 				}
 			}
 		}
-		stage('Perform unit tests') {
+		stage('Run microservices') {
 			steps {
 				script { echoBanner(STAGE_NAME) }
 				nvm(getNodejsVersion()) {
-					dir('./framework') { sh "npm run test:unit" }
-					dir('./services/core') { sh "npm run test:unit" }
+					sh 'pm2 start --silent ecosystem.jenkins.config.js'
 				}
+				waitForHttp('http://localhost:9901/api/ready')
+				// waitForHttp('http://localhost:9901/api/v2/blocks?timestamp=1615917187')
 			}
 		}
-		stage('Run microservices') {
+		stage('Perform unit tests') {
```

Comment:
As discussed previously, please separate the stages for running the unit tests (before the microservices are started) and functional tests (after the microservices are started).

Reply:
This change is already done with #464. I will pull the changes.